atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -239,6 +243,11 @@ boolean allSafeForLogging() {
         return false;
     }
 
+    @Value.Default
+    boolean useDialogue() {
```

Comment:
might want to call this `enableDialogue` so that if the atlasdb folks want to have some auto-reverting logic in here we couldn't get into the state where config said "useDialogue" but old clients were being used?